nbbuild/licenses/Apache-2.0-groovy-ant

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Is the `groovy-ant` artifact under the same license as groovy? If so, that can be modeled by adding a `Files` header to the `groovy-2.5.2-license.txt`: `Files: groovy-2.5.2.jar groovy-ant-2.5.2.jar`. In this case this file is not necessary.

If the licenses are different, then you need to add a `groovy-ant-2.5.2-license.txt` and `groovy-ant-2.5.2-notice.txt`.

Reply:
Both are from the same Groovy project, just that groovy-ant has no dependencies on ASM/ANTLR. Will use Files approach and update PR. Thanks for helping me figure out this stuff. Appreciated!